### PR TITLE
Updated README.md with OTEL environment variables

### LINKED ISSUE #17: Adding support for Aspire environment variables
Maybe setup OpenTelemetry-Target to recognize these Environment-Variables by default unless nothing else as been assigned:
- OTEL_EXPORTER_OTLP_ENDPOINT
- OTEL_EXPORTER_OTLP_PROTOCOL
- OTEL_EXPORTER_OTLP_HEADERS
- OTEL_RESOURCE_ATTRIBUTES

Where OTEL_EXPORTER_OTLP_ENDPOINT maps to Layout-property:
- Endpoint = `"${environment:OTEL_EXPORTER_OTLP_ENDPOINT}"`

Where OTEL_EXPORTER_OTLP_PROTOCOL can have following values:
- "http/protobuf" => OtlpProtocol.HttpProtobuf,
- "grpc" => OtlpProtocol.Grpc,
- _ => options.Protocol

Where OTEL_EXPORTER_OTLP_HEADERS + OTEL_RESOURCE_ATTRIBUTES must be parsed as comma-seperated mapping (`key1=value1;key2=value2`):
```
        foreach (var part in config?.Split(',') ?? [])
        {
            if (part.Split('=') is { Length: 2 } parts)
                resourceAttributes.Add(parts[0], parts[1]);
        }
```

### PR DESCRIPTION
**Pull request by snakefoot**
Resolves #17

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,11 +44,11 @@ Example XML config:
 
 ### Configuration parameters
 
-- **UseHttp** : Determines whether logs are exported using gRPC or HTTP. Optional, by default false, meaning the gRPC exporter will be used.
-- **Endpoint** : Determines where the exporter should send logs. Used to set OtlpExporterOptions.Endpoint. 
-If left empty, the default values will be used (optional)
-- **Headers** : Used to set OtlpExporterOptions.Headers (optional)
-- **ServiceName** : Used to set the service.name resource (optional)
+- **Endpoint** : Determines where the exporter should send logs. Used to set OtlpExporterOptions.Endpoint. When not specified it will use environment-variable: `OTEL_EXPORTER_OTLP_ENDPOINT`
+- **UseHttp** : Determines whether logs are exported using gRPC or HTTP. (optional). When not specified it will use environment-variable: `OTEL_EXPORTER_OTLP_PROTOCOL`
```
Comment (juliuskoval) [inline README.md:48]: I'd formulate the second sentence as 'When not specified, the protocol will be determined by the environment-variable: `OTEL_EXPORTER_OTLP_PROTOCOL`'. Also I'd mention that it is false by default

File: README.md [hunk 2]
```diff
@@ -44,11 +44,11 @@ Example XML config:
...
+- **Headers** : Used to set OtlpExporterOptions.Headers (optional). When not specified it will use environment-variable: `OTEL_EXPORTER_OTLP_HEADERS`
+- **ServiceName** : Used to set the service.name resource (optional). When not specified it will use environment-variable: `OTEL_SERVICE_NAME`
+- **UseDefaultResources** : Exclude default ressource like: telemetry.sdk.name, telemetry.sdk.language and telemetry.sdk.version. When not specified it will use environment-variable: `OTEL_RESOURCE_ATTRIBUTES`
```
Comment (juliuskoval) [inline README.md:51]: This isn't quite true. I'd rewrite it like this:
Use default resources like: telemetry.sdk.name, telemetry.sdk.language and telemetry.sdk.version. By default true.
If set to true, resources will be determined by the environment variable OTEL_RESOURCE_ATTRIBUTES. Setting any resources manually wil override the environment variable.

**Review by juliuskoval — CHANGES_REQUESTED** [review]
a few changes

**Comment by snakefoot** [comment]
Tried to improve the wording.

**Comment by juliuskoval** [comment]
Merged.

**Pull request merged**